doc/adr/0002-decouple-etl-from-mongodb-image-creation.md

```diff
@@ -0,0 +1,20 @@
+# 2. Decouple ETL from MongoDB image creation
```

Comment:
I don't think this needs to be here. If you look at what this repo is - a container with some data running MongoDB.
It would be good to know where that data comes from and the best place for that would be the `README`. Providing a link to the repo that does the ETL would be useful.